File: Beginner/1065 (Even Between five Numbers)/1065.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            int acm = 0;
            
            for(int i = 0; i < 5; i++)
            {
                int x = int.Parse(Console.ReadLine());
                if(x % 2 == 0)
                {
                    acm++;
                }
               
            }

            Console.WriteLine(acm + " valores pares");
      
        }
    }
}

File: Beginner/1065 (Even Between five Numbers)/1065.cpp
#include <iostream>
using namespace std;

int main(){
   int x[5], contP = 0 ;
   int i = 0 ;

   for (i = 0; i < 5; i++){
       cin >> x[i];
       if(x[i] % 2 == 0){
           contP++;
       }
   }

    cout << contP << " valores pares" << endl;
    system ("pause");
    return 0;
}